Sources/ForemWebView/ForemWebView.swift

```diff
@@ -40,42 +40,36 @@ open class ForemWebView: WKWebView {
     }()
 
     required public init?(coder: NSCoder) {
-        super.init(coder: coder)
+        let customConfig = ForemWebView.configuration()
+        super.init(frame: UIScreen.main.bounds, configuration: customConfig)
         setupWebView()
     }
 
     public override init(frame: CGRect, configuration: WKWebViewConfiguration) {
-        super.init(frame: frame, configuration: configuration)
+        let customConfig = ForemWebView.configuration(base: configuration)
+        super.init(frame: frame, configuration: customConfig)
         setupWebView()
     }
+    
+    // Helper function that helps recreate a custom configuration required before instantiation (init methods)
+    class func configuration(base configuration: WKWebViewConfiguration = WKWebViewConfiguration()) -> WKWebViewConfiguration {
+        let version = Bundle.main.infoDictionary?["CFBundleShortVersionString"] as? String
+        let frameworkIdentifier = "ForemWebView/\(version ?? "0.0")"
+        configuration.applicationNameForUserAgent = frameworkIdentifier
 
-    func setupWebView() {
-        // This approach maintains a UserAgent format that most servers & third party services will see us
-        // as non-malicious. Example: reCaptcha may take into account a "familiarly formatted" as more
-        // trustworthy compared to bots thay may use more plain strings like "Forem"/"DEV"/etc
-        // https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/User-Agent
-        evaluateJavaScript("navigator.userAgent") { (result, error) in
-            let version = Bundle.main.infoDictionary?["CFBundleShortVersionString"] as? String
-            let frameworkIdentifier = "ForemWebView/\(version ?? "0.0")"
-            if let result = result {
-                self.customUserAgent = "\(result) \(frameworkIdentifier)"
```

Comment:
So what will the report UserAgent of `ForeWebView` be now?

Reply:
Funny thing is: The UserAgent will actually be the same as before.

This code that is being removed evaluated `navigator.userAgent` and appended `ForemWebView/x.y` to it, so it acts as ["UserAgent extension"](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/User-Agent#syntax). The objective was to always use `WKWebView`'s default user agent (same as Safari) but append that information so it's available to the web app.

Turns out that's exactly what [`configuration.applicationNameForUserAgent`](https://github.com/forem/ForemWebView-ios/pull/35/files#diff-ea85a4cf11f0ff7ae7d6a3376844bd0ff0f78eb14c3395da8ca1607bd79f151cR58) does (it adds whatever you assign at the end of the UserAgent). [Apple docs](https://developer.apple.com/documentation/webkit/wkwebviewconfiguration/1395665-applicationnameforuseragent) might be a little vague, but I found this [SO answer](https://stackoverflow.com/a/50910170/3462026) explains it much better.

What matters is that `ForemWebView` appears in the UserAgent and we do have a check for it [in the tests](https://github.com/forem/ForemWebView-ios/pull/35/files#diff-60f724dcfbe827b8d295e0bf2df28176d15bc0ae230ee6fc8120da0d0339b5b6R46) 😃